decred/decred/dcr/dcrdata.py

```diff
@@ -861,7 +852,7 @@ def saveBlockHeader(self, header):
         self.heightMap[header.height] = bHash
         self.headerDB[bHash] = header
 
-    def sendToAddress(self, value, address, keysource, utxosource, feeRate=None):
+    def sendToAddress(self, value, address, keysource, utxosource, relayFee):
```

Comment:
If we're not going to have a default, let's add an optional argument with default value `allowHighFees=False` and then check that the fees aren't outrageous.